src/app/rpmostree-internals-builtin-start-daemon.c

```diff
@@ -292,42 +303,21 @@ connect_to_peer (int fd)
 
 
 int
-main (int argc,
-      char **argv)
+rpmostree_internals_builtin_start_daemon (int             argc,
+                                          char          **argv,
+                                          GCancellable   *cancellable,
+                                          GError        **error)
 {
-  GError *error;
-  GOptionContext *opt_context;
+  int exit_status = EXIT_FAILURE;
+  GOptionContext *opt_context = g_option_context_new ("rpm-ostreed -- rpm-ostree daemon");
   GIOChannel *channel;
   guint name_owner_id = 0;
-  gint ret;
-
-  ret = 1;
-  loop = NULL;
-  opt_context = NULL;
-
-  #if !GLIB_CHECK_VERSION(2,36,0)
-  g_type_init ();
-  #endif
-
-  /* See glib/gio/gsocket.c */
-  signal (SIGPIPE, SIG_IGN);
-
-  /* avoid gvfs (http://bugzilla.gnome.org/show_bug.cgi?id=526454) */
-  if (!g_setenv ("GIO_USE_VFS", "local", TRUE))
-    {
-      g_printerr ("Error setting GIO_USE_GVFS\n");
-      goto out;
-    }
-
+  
   opt_context = g_option_context_new ("rpm-ostreed -- rpm-ostree daemon");
```

Comment:
This is already set above.

Also, should that string be changed to simply e.g. `- rpm-ostree daemon` ?